rhobs/alerting/data_plane/prometheus.gitops_deployment_alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-deployment-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_deployment_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsDeploymentHealthErrors
+      expr: |
+        (
+          sum(kube_deployment_status_replicas_ready{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        )
+        /
+        (
+          sum(kube_deployment_spec_replicas{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        ) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
```

Comment:
I suggest adding the source_cluster to the summary/description

Reply:
Added it, PTAL.